tests/test_handlers.py

```diff
@@ -43,7 +43,8 @@ def test_all_handlers(input_dir: Path, output_dir: Path, tmp_path: Path):
         root=input_dir,
         path=input_dir,
         extract_root=tmp_path,
-        max_depth=DEFAULT_DEPTH,
+        max_depth=10,
+        entropy_depth=1,
```

Comment:
Since we are just giving the `__input__` dir to this test, I don't think the code will do anything with `entropy_depth=1`.
I hope it won't make things (so much) slower, but I would pass the same number to `max_depth` and `entropy_depth` for the tests.

Reply:
Hmm, actually thinking about this a bit more, I would just turn off entropy calculation for the integration tests, as we don't need it at all.